### PR TITLE
User flow

### PR DESCRIPTION
**Pull request by yashovardhan**
Add user flow page
Add whitelabel new docs
Add web3auth for dapps and wallets

### CODE REVIEW

File: docs/overview/web3auth-for-wallets.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+that for you. You can even save additional private keys safely within your application with Web3Auth.
+
+Using Web3Auth for your own wallet flows has the following advantages:
+
+1. The wallet owns the whole experience, with minimal external branding. You can even surpass that (have **zero Web3Auth Branding**) if you use
+   [Web3Auth Self Host](/self-hosting).
+2. You're even able to use your own OAuth 2.0 / Single Sign-On or any other flow for validating your user. You just need to provide Web3Auth with a
+   JWT `id_token` and it will handle the key generation for you.
+3. You can create your wallet for any chain, depending on your needs. Web3Auth generates the private key which can be used for any chain.
+4. You can even use Web3Auth to create an HDKey and generate multiple sub accounts for your users. This is useful for wallets that support multiple
```
Comment (shahbaz17) [inline docs/overview/web3auth-for-wallets.mdx:31]: What's HDKey?

Comment (yashovardhan) [reply docs/overview/web3auth-for-wallets.mdx:31]: I'm referring to this: https://www.npmjs.com/package/hdkey

Comment (shahbaz17) [reply docs/overview/web3auth-for-wallets.mdx:31]: Thanks for sharing. This was new info for me 😄

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Pull request merged**